.github/workflows/windows-build-and-test.yaml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - prerelease_test
   pull_request:
+defaults:
+  run:
+    shell: cmd
```

Comment:
Do we really want to use `cmd` instead of Powershell? It seems to have many drawbacks, like no good fail-fast mode.

Reply:
I don't have strong opinion of which shell it is better to use. I don't see we use anything special for PowerShell, but I might miss it. I haven't found what is the default in the docs, so  I am experimenting to see how to go ever the failure.

Reply:
I found that the default is powershell.